BUILD

```diff
@@ -8,14 +8,14 @@ cc_library(
   name = "nanopb",
   visibility = ["//visibility:public"],
   hdrs = [
-    "pb.h",
-    "pb_common.h",
-    "pb_decode.h",
-    "pb_encode.h",
+    "src/pb.h",
```

Comment:
It looks like I broke a bunch of things by moving these files.

Can you give me some direction as to how to fix them?